src/pages/Controllers/index.js

```diff
@@ -139,6 +140,28 @@ class Controllers extends Component {
     });
   };
 
+  isLoggedInUser = () => {
```

Comment:
Is there way to avoid repeating this code (DRY principle)?

Reply:
Yes, there is! The way I can think of is using a global object/method that keeps record of whether a user is logged in and then accessing it directly in the child components? In our design, the component that can serve this is `BasicLayout`

Reply:
+1, have you also considered moving this to the `utils` directory as a simple helper function?

Reply:
@aquibbaig ?

Reply:
Hey @gurbirkalsi @portante, yes! A simple helper function would help. Also, I probably need to convert BasicLayout to a functional component, if we do that we won't be able to use @connect decorator like we usually do. So, we need to move that to the end of the file where we export the component, something like a 'MapStateToProps' function would do.

Reply:
Why would BasicLayout need to be converted into a functional component to use the helper function? My impression is we would import the helper function into the relevant page component (in this case `Controllers/index.js`)

Reply:
Hey @gurbirkalsi, I actually tried using the `useContext` hook, which requires a functional component. The other way around is to use legacy context.